Validation/RecoMuon/plugins/NewMuonTrackValidatorBase.h

```diff
@@ -0,0 +1,281 @@
+#ifndef NewMuonTrackValidatorBase_h
```

Comment:
Adding a new "base" class is a good sign that inheritance here is unneeded. Lets remove it.